landing-page/content/common/release-notes.md

```diff
@@ -70,8 +70,25 @@ To add a dependency on Iceberg in Maven, add the following to your `pom.xml`:
 
 ## 1.1.0 release
 
+Has been released on 28 November 2022. The 1.1.0 release deprecates various pre-1.0.0 methods:
```

Comment:
I also remembered now that we can collect highlights from our iceberg sync document's highlights as @rdblue does a really good job of tracking them. 

Here are highlights that are missing from the current list. (I don't have tracking PR though, need to find it based on author's name)
Spark: Aggregate expressions were added (Thanks, Huaxin!)
Core: Added changelog table and new scan class (Thanks, Anton!)
Core: Added support for append/delete to branches (Thanks, Amogh!)
Core: Improved snapshot expiration (Thanks Szehon and Amogh!)
SnapshotProducer supports branch writes (Thanks, Namratha!)
FLIP-27 reader is supported in SQL (Thanks, Steven!)
Spark FunctionCatalog was added (Thanks, Kyle!)

Reply:
Collecting and exposing scan metrics (by @nastra) was also introduced in 1.1.0 AFAIK.

Reply:
@ajantha-bhat thanks, that's a great suggestion. I went through the list and added them with the links to the PRs. 

@gaborkaszab Thanks for the suggestion, the PR to scan metrics is already in the list :)